sigs/cncf-sigs.md

```diff
@@ -0,0 +1,250 @@
+
+* Clarify relationship between projects, CNCF project staff, and community volunteers.
+
+* Engage more communities and create an on-ramp to effective TOC contribution & recognition.
+
+* Reduce some project workload on TOC while retaining executive control & tonal integrity with this elected body.
+
+* Avoid creating a platform for politics between vendors.
+
+## Introduction
+
+A CNCF SIG will oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects.  Examples of such areas include security, testing, observability, storage, networking, etc.  The area overseen by a SIG is typically met by a set of CNCF projects, and may also represent a cross-cutting feature group shared by several projects (like security and observability).  SIG’s are:
+
+* long lived groups that report to the Technical Oversight Committee
+
+* led primarily by recognised experts in the relevant field(s), supported by other contributors
+
+CNCF SIGs are modelled on Kubernetes SIGs.  Differences are intended to be minimal to avoid confusion - unavoidable differences are described [here](https://docs.google.com/document/d/1oSGhx5Hw7Hs_qawYB46BvRSPh0ZvFoxvHx-NWaf5Nsc/edit?usp=sharing).
+
+## Responsibilities & Empowerment of SIGs
+
+It is the desire of the TOC that the CNCF SIGs, under guidance from the TOC, provide high-quality technical expertise, unbiased information and proactive leadership within their category.  The TOC makes use of this input to act as an informed and effective executive board to select and promote appropriate CNCF projects and practices, and to disseminate high quality information to end users and the cloud-native community in general.  SIGs explicitly have no direct authority over CNCF projects. In particular, the creation of CNCF SIG’s does not change the existing, successfully practiced [charter](https://github.com/cncf/foundation/blob/master/charter.md) goal that "Projects.. will be ‘lightly’ subject to the Technical Oversight Committee". 
+
+The SIGs should strive to present the TOC with easily understandable and votable "propositions", each of which is supported by clear written evidence.  A proposition may be “to approve this project for incubation based on this [written ](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md)[due diligence](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md)[ investigation](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md)”, or “to approve this landscape document based on these clear goals and evidence that it achieves them”.  It is of utmost importance that the information and proposals provided to the TOC by SIGs be highly accurate and unbiased, driven by the goal to improve the CNCF as a whole, rather than benefit one project or company over another.  We believe that the rising tide lifts all boats, and that is our goal.
```

Comment:
due-diligence-guidelines link repeated a few times, perhaps a text editor hiccup?

I've attempted to fix this and another minor formatting issue via PR to @quinton-hoole-2 branch: https://github.com/quinton-hoole-2/toc/pull/1

Reply:
Thanks @ultrasaurus .  I'll fold those in.